mache/cime_machine_config/config_machines.xml

```diff
@@ -206,6 +206,7 @@
         <command name="unload">aocc</command>
         <command name="unload">cudatoolkit</command>
         <command name="unload">climate-utils</command>
+        <command name="unload">matlab</command>
```

Comment:
Ah, I missed that one!  We need to include that in the `.sh` and `.csh` files for `pm-cpu`.  It wouldn't be the end of the world if we forget but it would be good.  It looks like we need to make some other updates, too, that got missed in the recent past.

https://github.com/E3SM-Project/mache/blob/main/mache/spack/pm-cpu_gnu_mpich.sh#L11

This becomes:
```
module rm cudatoolkit &> /dev/null
module rm climate-utils &> /dev/null
module rm matlab &> /dev/null
module rm craype-accel-nvidia80 &> /dev/null
```
Since it doesn't apply to any specific compiler or MPI library, it needs to apply to all the `pm-cpu*.sh` and `pm-cpu*.csh` files.

Reply:
Great, let me add those changes.